scripts/popup.js

```diff
@@ -23,7 +23,42 @@ function view_all_function(){
 	document.location.href = open_url;
 }
 
+function facebook_function(){
+	var link = "https://www.facebook.com/sharer/sharer.php?u=";
+	chrome.tabs.getSelected(null,function(tab) {
+		chrome.windows.create({'url': link+tab.url, 'type': 'popup', 'width' : 500, 'height' : 400}, function(window) {
+		});
+	});
+}
+
+function google_function(){
+	var link = "https://plus.google.com/share?url=";
+	chrome.tabs.getSelected(null,function(tab) {
+		chrome.windows.create({'url': link+tab.url, 'type': 'popup', 'width' : 500, 'height' : 400}, function(window) {
+		});
```

Comment:
Your welcome :) @collinx 
i mean to say that search is done on wayback machine's archives and those results will be available to us . so as the results from this needed to be shared on social media.
In short the url that we are going to share is url of search in conjunction with wayback machine 

I hope it helps :)